src/client/pages/EmailSent.tsx

```diff
@@ -168,28 +167,15 @@ export const EmailSent = ({
 						submitButtonText="Submit verification code"
 						disableOnSubmit
 					>
-						<TextInput
-							label="Verification code"
-							type="text"
-							width={4}
-							pattern="\d{6}"
-							name="code"
-							autoComplete="one-time-code"
-							inputMode="numeric"
-							maxLength={6}
-							error={
-								fieldErrors?.find((error) => error.field === 'code')?.message
-							}
-							defaultValue={passcode}
-						/>
+						<PasscodeInput passcode={passcode} fieldErrors={fieldErrors} />
 					</MainForm>
 				</div>
 			)}
 			<InformationBox>
 				<InformationBoxText>
-					Didn’t get the email? Check your spam
+					Didn’t get the email? Check your spam&#8288;
```

Comment:
ooh so this is a word joiner?

https://www.codetable.net/decimal/8288

Reply:
Yes, it's mildly cursed, welcome to 2024 we still need to do weird Unicode tricks